libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts

```diff
@@ -292,39 +300,46 @@ function addCoreBot(services: ServiceCollection, configuration: Configuration):
     services.addFactory<
         ActivityHandlerBase,
         {
+            botFrameworkAuthentication: BotFrameworkAuthentication;
             botTelemetryClient: BotTelemetryClient;
             conversationState: ConversationState;
+            dialogs: Dialog[];
             memoryScopes: MemoryScope[];
             pathResolvers: PathResolver[];
             resourceExplorer: ResourceExplorer;
             skillClient: SkillHttpClient;
             skillConversationIdFactory: SkillConversationIdFactoryBase;
             userState: UserState;
+            languagePolicy: LanguagePolicy;
         }
     >(
         'bot',
         [
+            'botFrameworkAuthentication',
             'botTelemetryClient',
             'conversationState',
+            'dialogs',
             'memoryScopes',
             'pathResolvers',
             'resourceExplorer',
             'skillClient',
```

Comment:
Isn't this dependency no longer required since the AdaptiveDialogBot uses `BotFrameworkAuthentication.createBotFrameworkClient()`?